app/containers/Applications/ApplicationDetailsCardItem.tsx

```diff
@@ -43,11 +46,27 @@ export const ApplicationDetailsCardItemComponent: React.FunctionComponent<Props>
     >
       <Card.Header onClick={() => setIsOpen(!isOpen)}>
         <Row>
-          <Col md={9}>
-            <h2>{formJsonByFormId.name}</h2>
+          <Col md={6}>
+            <h4>{formJsonByFormId.name}</h4>
           </Col>
-          <Col md={3} style={{textAlign: 'right'}}>
-            <Button>{isOpen ? 'Expand' : 'Collapse'}</Button>
+          {isAnalyst ? (
+            <Col style={{textAlign: 'right'}}>
```

Comment:
The card expanding/collapsing is done by a click handler on the header. This will cause any click in the `ApplicationReviewContainer` to propagate up and toggle the card. 
You can either move the click handler to the button defined below or add a click handler here that stops click event from propagating.